app/models/concerns/host_params.rb

```diff
@@ -20,13 +20,15 @@ def clear_host_parameters_cache!
     def host_inherited_params(include_source = false)
       hp = {}
       params = host_inherited_params_objects
+      renderer = SafeRender.new(:variables => { :host => self })
```

Comment:
@ShimShtein for future reference, I think it would make sense to store on params (global, class etc) if the value has <% in it or not, this can probably save a few ms instead of calling the SafeRender by default.